scripts/loadtest-scripts/clients/sdcclient.py

```diff
@@ -0,0 +1,25 @@
+from clients import http
+from clients.actionclient import ActionClient
+from clients.collectionexerciseclient import CollectionExerciseClient
+from clients.http import factory
+
+
+class SDCClient:
```

Comment:
Where else do you propose we do the injection? If we do it inline in the script then it will need to be replicated where ever these clients are used. Also, this provides a nicer DSL IMHO.